src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:35411/organizations/org3/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:35411/project/issues?id=sample&pullRequest=6386&issues=AWkW-_xr7Fn9JAuMjTwh&open=AWkW-_xr7Fn9JAuMjTwh)

[ISSUE_KEY]: AWkW-_xr7Fn9JAuMjTwh